### DISCUSSION TITLE
let the map wrap east-west

### DISCUSSION THREAD

**Discussion by jhnc** [organicmaps/organicmaps#3068] [Ideas]
(Sorry if this is addressed in a topic I didn't find.)

The world is a globe. In organicmaps it is presented as a cylinder, which is not unreasonable (this isn't google earth). However, viewing anywhere along the international dateline is very tedious.

For example:  zoom in / go east until Fiji is displayed. Find Vuna Village. Okay, now go east and a bit north to bring Wainibau Waterfalls onscreen. Oops - can't get far enough east. Have to zoom out, go all the way west around the world and then zoom back in again. And you can't get to Tonga, Samoa, etc, by going north-east either.

One should be able to head eastward from China and get to Hawaii and then head west from there and get to Malaysia, the same way my ancestors did. :-)

Google Maps allows this. Bing Maps allows it. The Openstreetmaps website allows it (albeit showing me 8 copies of South America if I zoom out too much). Even the map on my bike computer app allows it.

This seems like functionality that organicmaps should have.

**Comment by biodranik** [comment]
See #1373

**Reply by jhnc** [reply]
d'oh. I think I only searched the discussions, not the issue queue! thanks
